tests/test_coords.py

```diff
@@ -36,5 +39,11 @@ def test_data_destriding(self):
         )
 
 
+def test_multipoint_ctypes_deprecated():
+    coords = geometry.LineString([[12, 34], [56, 78]]).coords
+    with pytest.warns(ShapelyDeprecationWarning, match="ctypes"):
+        coords.ctypes
```

Comment:
@jorisvandenbossche would the following suffice?

```suggestion
    with pytest.warns(ShapelyDeprecationWarning, match="ctypes"):
        geometry.LineString().coords.ctypes
```

I feel like the unused coordinate values are slightly distracting. Also, what would you think about a minimal docstring for the test?

Reply:
Added a docstring (and also fixed the test name, that was a copy/paster error).

`geometry.LineString().coords.ctypes` actually doesn't work, because for an empty geometry the `coords` attribute returns an empty list instead of a CoordinateSequence object (to me this feels a bit like a bug, but is done on purpose in the code).

Reply:
I agree about the bug.